src/helpers/featureFlags.ts

```diff
@@ -33,22 +33,34 @@ export function setConfigFromFeatureFlags(config: IChartConfig, featureFlags: IF
     return result;
 }
 
-export function getTableConfigFromFeatureFlags(featureFlags: IFeatureFlags): IPivotTableConfig {
+export function getTableConfigFromFeatureFlags(
+    config: IPivotTableConfig,
+    featureFlags: IFeatureFlags,
+    predicateEnvironment: boolean = true,
```

Comment:
This parameter name is a bit confusing. What does it do? Why does it only affect `enableTableColumnsGrowToFit`?

Reply:
renamed. GrowToFit is turned on only on Dashboards. In AD and URI Visualizations it is turned off without influence of FF